db/seeds.d/03-permissions.rb

```diff
@@ -1,169 +1,3 @@
-permissions = [
```

Comment:
Moving to separate file in seeds.d does not work for me, I need the lists available in migrations as well.